<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Center</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="center_color">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Orange</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Yellow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Green</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sky</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Blue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Purple</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>White</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Saturation</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="center_saturation">
            <item>
             <property name="text">
              <string>High</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Med</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Low</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Null</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Left</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="left_color">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Orange</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Yellow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Green</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sky</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Blue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Purple</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>White</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Saturation</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="left_saturation">
            <item>
             <property name="text">
              <string>High</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Med</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Low</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Null</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Right</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="right_color">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Orange</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Yellow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Green</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sky</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Blue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Purple</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>White</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Saturation</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="right_saturation">
            <item>
             <property name="text">
              <string>High</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Med</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Low</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Null</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Mode</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>&lt;b&gt;Keyboard&lt;/b&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="mode">
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gaming</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Breathe</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Demo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wave</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
